### DISCUSSION TITLE
skywalking dashboard K8s

### DISCUSSION THREAD

**Discussion by chenqinwei** [apache/skywalking#8007] [Q&A] [answered]
### Search before asking

- [x] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

What data are displayed in k8s and VM of skywalking dashboard? How did these data come from? How to use this function？
![image](https://user-images.githubusercontent.com/88814006/138814429-847f2ce0-a1f2-4a16-921b-69583930092b.png)

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment, marked as answer]
As always, read document first before submitting issues or discussions.

https://skywalking.apache.org/docs/main/v8.8.1/en/setup/backend/backend-vm-monitoring/
